test/units/galaxy/test_api.py

```diff
@@ -1353,3 +1353,11 @@ def test_clear_cache(cache_dir):
 def test_cache_id(url, expected):
     actual = galaxy_api.get_cache_id(url)
     assert actual == expected
+
+
+def test_should_retry_error_with_invalid_code(monkeypatch):
+    expected = "Invalid value for HTTP retry code"
```

Comment:
Since it's a regex, it'd be useful to be extra explicit about it:
```suggestion
    expected = r"^Invalid value for HTTP retry code$"
```